doc/040_backup.rst

```diff
@@ -229,9 +229,11 @@ see `filepath.Match <https://golang.org/pkg/path/filepath/#Match>`__ for
 syntax. Patterns are tested against the full path of a file/dir to be saved,
 even if restic is passed a relative path to save.
 
-Environment-variables in exclude files are expanded with `os.ExpandEnv <https://golang.org/pkg/os/#ExpandEnv>`__,
+Environment variables in exclude files are expanded with `os.ExpandEnv <https://golang.org/pkg/os/#ExpandEnv>`__,
 so ``/home/$USER/foo`` will be expanded to ``/home/bob/foo`` for the user ``bob``.
-To get a literal dollar sign, write ``$$`` to the file. Note that tilde (``~``) expansion does not work, please use the ``$HOME`` environment variable instead.
+To get a literal dollar sign, write ``$$`` to the file - this has to be done even
+there's no matching environment variable for the word following a single ``$``.
+Note that tilde (``~``) is not expanded, instead the ``$HOME`` environment variable.
```

Comment:
Missing verb. Maybe

> Note that tilde (``~``) is not expanded to the user's home directory like in some Unix tools. Use the ``$HOME`` environment variable instead.

Reply:
Thanks a lot for noticing that missing word :D I think we'll keep the text just "is not expanded", because that's the relevant part of the info, and if we make it according to your suggestion, this implies that the tilde *may* be expanded, just not to the user's home directory, which leaves uncertainty.